extensions/positron-r/package.json

```diff
@@ -155,15 +155,6 @@
             "default": false,
             "description": "%r.configuration.restoreWorkspace.description%"
           },
-          "positron.r.extraArguments": {
-            "scope": "window",
```

Comment:
While we're here, I also wonder if we should universally remove or universally include the `"scope"`. `"window"` is the default, FWIW, and all we currently use in positron-r. Trying to figure out what that meant and whether I should include it or not is actually what lead to the scrutiny 🧐 where I found this duplication.

Reply:
I certainly cannot find or think of a reason to be explicit with `"scope": "window"`, given that it is the default and we don't set any other scope level in the whole extension.

If folks want to peruse more:
<https://code.visualstudio.com/api/references/contribution-points#contributes.configuration>

Reply:
I'm not going to make additional changes in this PR. But perhaps we should rationalize this at some point. I almost did it here, then I started to wonder if maybe we *should* be using other scopes more than we do 🤔. Food for future thought.